proto/serverless/proto/instrumentation/tagset/v1/tagset.proto

```diff
@@ -0,0 +1,272 @@
+syntax = "proto3";
+
+package serverless.proto.instrumentation.tagset.v1;
+
+message Tags {
+  // The required set of tags for The Serverless Ingest Platform. Payloads without all these tags will be rejected.
+  SlsSdkTags sls = 1;
+
+  // ============================================ //
+  // Defined TagSets start at field number 4000  //
```

Comment:
Why start at 4000?

Reply:
No particular reason, more so just to have a clear separation between groups of fields.

Reply:
Wouldn't it be more reasonable (easier for maintenance) to keep this separation number lower, e.g. `100` ?

Reply:
I don't think it really matters. We can change it, was just following conventions I've used in the past.

Reply:
I think 100 would make a friendlier base,  unless there was some deeper thought behind `4000` I'm not aware of.